# Disclaimer

_** Luis Perez is not responsible for any errors or omissions, or for the results obtained from the use of the information in this document. All information in this document is provided "as is", with no guarantee of completeness, accuracy, timeliness, or of the results obtained from the use of this information, and without warranty of any kind, express or implied. **_

_** The views, opinions, information, and findings expressed in this document are those of the author and do not necessarily reflect the official policy or position of Benedictine University. Any content provided in this document by the author is of his/her opinion and is not intended to malign any religion, ethnic group, club, organization, company, individual, or anyone or anything. The findings and information in this document are not representative of Benedictine University students nor of Benedictine University. **_



<br>
<hr style="border:1px solid gray"> </hr>

<br>

# Calculating sample size
Calculating the sample size of an infinte population  

$n_{0} = \frac{Z^{2}pq}{e^{2}}$

Z represents z score, p is population proportion, q is 1-p, e is the confidence interval

Calculating the sample size of a finite population

$n =  \frac{n_{0}}{1+\frac{n_{0}-1}{N}}$

N represents the target population size


    
    

In [1]:
def sample_size(confidence_level,population_proportion,confidence_interval,population_size):
    from statistics import NormalDist

    alpha = 1-confidence_level
    z_score = NormalDist().inv_cdf(1-alpha/2) 
    n_subscript_zero=(z_score**2*population_proportion*(1-population_proportion))/confidence_interval**2
    n =  n_subscript_zero/(1+(n_subscript_zero-1)/population_size)
    return round(n)
sample_size(.95,.5,.05,450) # confidence level is 95%, population proportion is unknown, a conservative estimate value of .5 is chosen, confidence interval ±5%, the approximated size of the population is 450


207

207 randomly chosen partipants would be the adequate sample size. Unfortunately I do not have acces to the emails of the all students of Benedictine University. I instead sent out the survey to people in my classes, approximately 80 Benedictine University students, only 28 responded to the survey.  